Mohammed/index.js

```diff
@@ -0,0 +1,45 @@
+
+// 0.
+let apples = 20;
+let orange = 30;
+
+// 1.
+let comp = apples == orange ? "They are equal" : "They are not equal";
+console.log(comp);
+
+// 2.
+comp = apples != orange ? "They are not equal" : "They are equal";
+console.log(comp);
+
+// 3.
+comp = apples > orange ? "Apple is greater than Orange" : "Orange is greater than Apple";
+console.log(comp);
+
+// 4.
+comp = apples <= orange ? "Apple is smaller than or equals to Orange" : "Orange is smaller than or equals to Apple";
+console.log(comp);
+
+// 5.
+comp = orange > apples ? "Orange is greater than Apple" : "Orange is not greater than Apple";
+console.log(comp);
+
+// 6.
+let mangos = 5;
+comp = (mangos*apples) > (mangos+orange) ? "mangos * apples is greater than mangos + orange":"mangos*apples is not greater than mangos+orange";
```

Comment:
Please get rid of line 28! The answer is wrong. The correct answer is: 
console.log( (mangoes * apples) > (mangoes + oranges) );